components/script/dom/range.rs

```diff
@@ -343,7 +343,7 @@ impl RangeMethods for Range {
         } else {
             // Step 3.
             self.set_start(node, offset);
-            if !(self.start() <= self.end()) {
```

Comment:
It looks like this change needs to be reverted as it's causing a failure on the bots.

Reply:
> It looks like this change needs to be reverted as it's causing a failure on the bots.

Hello sir, let me revert it. 
Do the bots only allow this particular code?